CONTRIBUTING.md

```diff
@@ -79,13 +79,16 @@ Helpful issues usually include:
 
 Contributions to the documentation site are made by editing the docs repository. You can do this directly on GitHub.com or by creating a copy of the repository locally, making your changes there, and contributing back to our repository.
 
+> By default, your merged PR to an English page will trigger our Translation Status Tracker. If your change should **NOT** be applied to every language (e.g. a typo fix to an English word), please include the word "minor" in your PR title and/or in a commit message.  See the next section for more details.
```

Comment:
Do we want to avoid using `<blockquote>` for things that aren’t quotations?

If we do want to highlight this I guess we could use the GitHub syntax I suggested in #1078 here too:

```suggestion
> **Note**
> By default, your merged PR to an English page will trigger our Translation Status Tracker. If your change should **NOT** be applied to every language (e.g. a typo fix to an English word), please include the word "minor" in your PR title and/or in a commit message.  See the next section for more details.
```

I _think_ GitHub unfortunately still uses `<blockquote>` to render these, but given they’re detecting the custom syntax, I suppose there’s hope they might switch to better markup in the future.

Reply:
Yes, we *do* want to avoid `<blockquote>` -- I searched and couldn't find GitHub's new "note" notation, that I thought was of the form `:::note ` like ours, but it didn't preview well here. Does anyone know?

Reply:
Oh! It’s my suggestion! Here’s the page with details: https://github.com/github-community/community/discussions/16925